### DISCUSSION TITLE
Confused about template naming for MVC

### DISCUSSION THREAD

**Discussion by dotnetshadow** [Dotnet-Boxed/Templates#199] [Q&A]
Hi there,

First I like the new naming and branding of the project, well done.

I originally used ASP.NET Core MVC 6 Boilerplate VSIX when you first created this project.

I now see that you have 2 templates:
*ASP.NET Core API Boxed* and *ASP.NET Core GraphQL Boxed*.

So if I'm creating a new ASP.NET Core MVC application which template should I be using? Can either just be used ? 

To me the API almost sounds like it's good for web api type projects as opposed to just standard MVC apps? 

Am I misunderstanding something?

**Comment by RehanSaeed** [comment]
I don't have an MVC template right now. The reason being that the old MVC template was based on .NET Core 1, project.json and Gulp. Today, I'd build a template using .NET Core 2.1, csproj, Webpack and maybe even Blazor. There was just too much changing to keep up and I've got limited time. The web just changes too fast to keep up. I do want to bring this back though.

**Comment by dotnetshadow** [comment]
Cheers thanks for clearing it up, I do agree that it was a bit of a maintenance issue trying to keep things in sync. I'm glad you still have the taghelpers that are handy especially for social graphs.

**Comment by RehanSaeed** [comment]
Yes, I'll maintain NuGet packages with a view to using them when I have the time to build a new MVC/Blazor template.
